Formula/i/ipopt.rb

```diff
@@ -68,6 +68,10 @@ def install
         "libseq/#{shared_library("*")}",
         "PORD/lib/#{shared_library("*")}"
       ]
+      # install headers
+      libexec.install "include"
+      (libexec/"include").install Dir["libseq/*.h"]
+      include.install_symlink Dir[libexec/"include/*"]
```

Comment:
Why not just directly installing them into `include` instead of installing into `libexec` and then creating symlinks?

Reply:
In that case, why not install things into a `mumps` subdirectory?

Reply:
The naming really isn't that important to me (`mumps` vs `libexec` just so long as they're symlinked into `include`, I'll renamed to `mumps`.